### DISCUSSION TITLE
Sending messages

### DISCUSSION THREAD

**Discussion by tscorsel** [community/community#37243] [Other Feature Feedback, Questions, & Ideas] [answered]
How do you DM someone?

**Comment by byrneh** [comment, marked as answer]
Github doesn't support direct messenging between memeber accounts.

**Reply by JesseTheExplorer** [reply]
lol, thats just **bad** design

edit: jk

**Comment by jfullstackdev** [comment]
one thing you can do as a workaround is to make a private repo, invite that user, if he/she accepts that, then you can start messaging each other through the Discussion tab or Issues tab, only the two or maybe more than that can see those messages

**Comment by liderbug** [comment]
Well that sucks.  Person A had a problem (I have the same problem).  Person B answered the question somewhat cryptically.  Person A responded with "That didn't quite work, but I made a minor change and now it works".  WHAT WAS THE ANSWER? AND WHAT WAS THE MINOR CHANGE? -  Come on folks - Oh we don't allow users to communicate with each other
